actions/maven-publish/action.yml

```diff
@@ -15,7 +15,7 @@ runs:
     - name: Publish API package
       id: maven-publish
       run: |
-        ./gradlew publish
+        ./gradlew publishDefaultPublicationToGitHubPackagesRepository
```

Comment:
I'd rather not have publishing only work for GitHub. Using the `publish` command allows us to define any repository we like. 
[maven-publish documentation](https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:tasks)